tests/test_zmq_interface.py

```diff
@@ -0,0 +1,135 @@
+from dc_federated.backend.zmq_interface import ZMQInterfaceModel, ZMQInterfaceServer
+import zmq
+from unittest.mock import Mock, patch
+import threading
+import time
+import pytest
+
+# Initalise some mocks that can be used in place of the callbacks to check that
+# they are called. Along with some other constants that are used to check for
+# equality in the tests.
+register_worker_callback = Mock()
+unregister_worker_callback = Mock()
+GLOBAL_MODEL = {"test": "test_val"}
```

Comment:
can you try a value of the same type as the models instead of a plain string?

Reply:
Hmm, I considered that but then I thought it was a bit heavy handed for a test to load pytorch create a pytorch model and serialise and deserialise it. Can do if you want though. Or maybe there's a shortcut somehow to test sending something similar?